### ISSUE TITLE
Magnets in cubes for any BLD event

### ISSUE THREAD

**Issue by kingmathyall** [thewca/wca-regulations#402]
I saw this post by Weston Mizumoto on Speedsolving Forum and thought it would be best to bring more attention to what he brought up in his post:

"When putting magnets into a cube, one has to take care that the orientation of every pair magnets is correct. If you put a pair of magnets in reversed, then in many unsolved positions, some of the magnets will be repelling while others are attracting. This makes the solved position of the cube have a large amount of tactile feedback, while other positions of the cube have less tactile feedback. This means that if magnets are inserted "improperly", then if someone is solving BLD and they know that if the magnetic "clicks" in their current cube state are "weak", then they can determine that the cube is not solved."

tl;dr Magnets can help determine whether a cube is solved or not, if the magnets are placed in the right spots

While talking with my friends about this, one situation they brought up was the following:
>get pop doing BLD
>put piece in wrong
>notice it repels
>flip it around

After discussing with a well-respected delegate about the issue we both agreed that it should be added to the regulations that magnets cannot be used in any cube(s) used for any BLD event.

While it might be difficult to enforce because it is not a visual thing, magnetic cubes when turned can be identified. Also, dragging the cube along some paperclips and seeing if the paperclip sticks onto the cube is a pretty easy way to tell as depicted by the following image: http://bit.ly/2gtVtXO

Once again, I want to say that all credit goes to Weston Mizumoto for coming up with this "red flag" so to speak.

There are/were regulations about using certain types of cubes for BLD events before that I think sort of fall under a similar category: pillowed cubes & stickerless cubes
-I believe that magnetic cubes can be taken advantage of in an unfair manner if put into the right hands

Let me know your thoughts on this issue.

**Comment by Jambrose777** [comment]
I believe that this is already in our regulations under:
3j) Puzzles must be clean, and must not have any markings, elevated pieces, damage, or other differences that significantly distinguish any piece from a similar piece. 

This would be like a side being more loose than another. This is catchable when scrambling (if the defect is obvious enough) or if the competitor is turning the puzzle weirdly to determine the correctness of a piece (which can be caught the same as a loose side).

I'm a little confused on your interpretation of your example. On a perfectly magnetized cube, if you flip an edge, the edge feels the same.

**Comment by kingmathyall** [comment]
One could possibly place magnets in different places so that it feels the "strongest" when it is solved where is if it's still scrambled a little bit, one could possibly tell by turning a layer to feel if the magnetic pull is stronger or weaker than usual.

**Comment by KitClement** [comment]
Also, I think the other issue is if we can distinguish partially magnetic cubes from ones that are fully magnetic.  Assuming we cannot distinguish between the two, the conservative option would be to not allow magnets for BLD for their potential to create a tactile advantage.  We already disallow overlay logos due to their tactile advantage, and this seems to have some parallels to this issue.

Of course, I don't know how distinguishable these kinds of magnetic cubes would be, and thus I don't know how necessary it would be to take a conservative approach to this issue.

**Comment by Laura-O** [comment]
I think the comparison with overlay stickers is not suitable: you can easily feel a worn out overlay  sticker during a solve, but it's comparably difficult to identify non-magnetized pieces.

I have tried to do that with my Weilong GTSM and exchanged pieces with a regular GTS. It might be possible to identify pieces or layers, but it's not easy. As Jacob wrote, I think it's similar to having one side more loose than another and the moves you have to do to identify prepared pieces really look suspicious.
I cannot deny that there might be ways to do that better and of course to practice the recognition, but I currently do not think that this way to cheat is alarming and we need to take actions now.
